canonical-websites/www.ubuntu.com
# Updated short form agreement

## Done

- Updated short form agreement as per [copydoc](https://docs.google.com/document/d/1psOjmP69PFHs1rhDfP043kXksYieGlny9-0r6yWUPnI/edit)
- Updated Japanese short form agreement as per [copydoc](https://drive.google.com/open?id=0B2_PyaH0PaVmT290eFhPY2VDOWNLbXhva3NVbkZoWTlOT3NR)
- Updated 'Older versions' with old (current) agreement

## QA

- Check that /legal/short-terms matches copydoc in English and Japanese
- Check that links to old versions still work properly


## Issue / Card

Fixes #2276
